Genius/ContentPoster.cs

```diff
@@ -122,12 +117,24 @@ public class ContentPoster
         }
     }
 
+	/// <summary>
+	/// The payload that is contained in annotations
+	/// </summary>
     public class AnnotationPaylod
     {
+		/// <summary>
+		/// The annotation itself.
+		/// </summary>
         [JsonProperty(PropertyName = "annotation")]
         public Annotation Annotation { get; set; }
+		/// <summary>
+		/// The referent.
+		/// </summary>
         [JsonProperty(PropertyName = "referent")]
         public Referent Referent { get; set; }
+		/// <summary>
+		/// The web page.
```

Comment:
Again, No need to comment webpage on a variable named WebPage. It self-explains itself.